kore/src/Kore/Attribute/PredicatePattern.hs

```diff
@@ -97,6 +100,10 @@ isSimplified
     :: SideCondition.Representation -> PredicatePattern variable -> Bool
 isSimplified sideCondition = Simplified.isSimplified sideCondition . simplifiedAttribute
 
+isSimplifiedIgnoreCondition
```

Comment:
Please add documentation similar to `Kore.Attribute.Pattern.isSimplifiedIgnoreCondition`.